mixins/labelled-mixin.js

```diff
@@ -102,6 +100,9 @@ export const LabelledMixin = superclass => class extends superclass {
 
 		});
 
+		if (!labelElem) return;
+		const ancestor = getCommonAncestor(this, labelElem);
```

Comment:
The call to `getCommonAncestor` was throwing an error when `labelElem` was `null` -- which it was intentionally in one of the tests.